TombEngine/Objects/TR2/Entity/tr2_knife_thrower.cpp

```diff
@@ -264,7 +233,7 @@ namespace TEN::Entities::Creatures::TR2
 
 				if (!creature->Flags)
 				{
-					CreatureEffect(item, KnifeBiteLeft, ThrowKnife);
+					CreatureEffect2(item, KnifeBiteLeft, 100, torso, ThrowKnife);
```

Comment:
What's the difference between `CreatureEffect` and `CreatureEffect2`? If that's not an original Core naming, I'd introduce a bit more descriptive name for this function.

Reply:
CreatureEffect use item->Animation.Velocity.z for speed, CreatureEffect2 let the programmer specify it in the argument.